### ISSUE TITLE
Funders are coming in as "UNSPECIFIED"

### ISSUE THREAD

**Issue by rsdoiel** [caltechlibrary/eprinttools#44]
The eprint-import process is having funders identified as "UNSPECIFIED" even though a grant number is present. See record in production review buffer 112030

**Comment by rsdoiel** [comment]
See XML versioned imported. Many empty items are being generated.

```
<eprint>
  <eprintid>112030</eprintid>
  <rev_number>1</rev_number>
  <eprint_status>buffer</eprint_status>
  <userid>5487</userid>
  <importid>0</importid>
  <dir>disk0/00/11/20/30</dir>
  <datestamp>2021-11-23 16:26:23</datestamp>
  <lastmod>2021-11-24 00:40:45</lastmod>
  <status_changed>2021-11-23 16:26:23</status_changed>
  <type>article</type>
  <succeeds>0</succeeds>
  <commentary>0</commentary>
  <latitude>0</latitude>
  <longitude>0</longitude>
  <item_issues_count>0</item_issues_count>
  <sword_depositor>0</sword_depositor>
  <title>The SCUBA-2 web survey: I. Observations of CO(3–2) in hyper-luminous QSO fields</title>
  <ispublished>pub</ispublished>
  <full_text_status>none</full_text_status>
  <date>2019-05-01</date>
  <date_type>published</date_type>
  <publication>Monthly Notices of the Royal Astronomical Society</publication>
  <volume>485</volume>
  <number>1</number>
  <publisher>Royal Astronomical Society</publisher>
  <pagerange>753-769</pagerange>
  <pages>0</pages>
  <issn>0035-8711</issn>
  <related_url>
    <item>
      <url>http://academic.oup.com/mnras/article-pdf/485/1/753/28488455/stz429.pdf</url>
    </item>
  </related_url>
  <num_pieces>0</num_pieces>
  <doi>10.1093/mnras/stz429</doi>
  <funders>
    <item></item>
    <item></item>
    <item>
      <grant_number>2017YFA0402700</grant_number>
    </item>
    <item>
      <grant_number>ST/P000541/1</grant_number>
    </item>
    <item></item>
    <item></item>
    <item></item>
    <item></item>
    <item>
      <grant_number>17H04831</grant_number>
    </item>
    <item>
      <grant_number>17KK0098</grant_number>
    </item>
  </funders>
  <thesis_defense_date>0000-00-00</thesis_defense_date>
  <thesis_degree_date>0000-00-00</thesis_degree_date>
  <thesis_approved_date>0000-00-00</thesis_approved_date>
  <thesis_public_date>0000-00-00</thesis_public_date>
</eprint>
```

**Comment by rsdoiel** [comment]
Corrected in 1.1.1-rc2.
